queries.sql

```diff
@@ -1,60 +1,204 @@
+LIMIT 1;
+SELECT COUNT(animals.name)
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+WHERE vets.id = 3
+GROUP BY vets.name;
+SELECT vets.name AS Vets,
+    species.name AS Specialities
+FROM specializations
+    JOIN species ON specializations.species_id = species.id
+    RIGHT JOIN vets ON specializations.vets_id = vets.id;
+SELECT animals.name,
+    date_of_visit
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+WHERE vets.id = 3
+    AND date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+SELECT animals.name,
+    COUNT(animals.name)
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+GROUP BY animals.name
+ORDER BY COUNT(animals.name) DESC
+LIMIT 1;
+SELECT animals.name,
+    date_of_visit
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+WHERE vets.id = 2
+ORDER BY date_of_visit ASC
+LIMIT 1;
+SELECT animals.name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    vets.name AS vets_name,
+    vets.age AS vets_age,
+    vets.date_of_graduation AS vets_graduation,
+    date_of_visit
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+ORDER BY date_of_visit DESC
+LIMIT 1;
+SELECT COUNT(*)
+FROM visits
+    JOIN vets ON visits.vets_id = vets.id
+    JOIN animals ON visits.animals_id = animals.id
+    JOIN species ON animals.species_id = species.id
+    LEFT JOIN specializations ON vets.id = specializations.vets_id
+    AND species.id = specializations.species_id
+WHERE specializations.id IS NULL;
+SELECT species.name,
+    COUNT(species.name) AS speciality
+FROM vets
+    JOIN visits ON vets.id = visits.vets_id
+    JOIN animals ON visits.animals_id = animals.id
+    JOIN species ON animals.species_id = species.id
+WHERE vets.name = 'Maisy Smith'
+GROUP BY species.name
+ORDER BY speciality DESC
+LIMIT 1;
 
-SELECT name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE owner_id=4;
-SELECT animals.name FROM animals JOIN species ON animals.species_id=species.id WHERE species_id=1;
-SELECT name,full_name FROM animals RIGHT JOIN owners ON animals.owner_id=owners.id;
-SELECT species.name,COUNT(*) FROM animals JOIN species ON animals.species_id=species.id GROUP BY species.id;
-SELECT animals.name FROM animals JOIN owners ON animals.owner_id=owners.id JOIN species ON animals.species_id=species.id WHERE owner_id=2 AND species_id=2;
-SELECT name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE owner_id=5 AND escape_attempts=0;
-SELECT full_name,COUNT(owner_id) FROM animals JOIN owners ON animals.owner_id=owners.id GROUP BY full_name ORDER BY COUNT(owner_id) DESC LIMIT 1;
+CREATE INDEX idx_animals_id ON visits (animals_id);
+CREATE INDEX idx_vets_id ON visits (vets_id);
+CREATE INDEX idx_email ON owners (email);
```

Comment:
👍 Good progress on adding the indexes! However, there's a small adjustment to make to align with the curriculum guidelines.

- [ ] Currently, your index creation operation is inside the `queries.sql` file. According to the [curriculum-databases-projects-template](https://github.com/microverseinc/curriculum-template-databases#:~:text=This%20repository%20includes,Use%20them%20responsibly!), `CREATE` operations like index creation should be inside the `schema.sql` file. Kindly move these operations to the appropriate `schema.sql` file.

By organizing your SQL scripts this way, you'll ensure that your project structure aligns with the curriculum guidelines. Keep up the great work! 👍📜